content/docs/reference/cli-set-context.md

```diff
@@ -0,0 +1,125 @@
+├──────────────────────────┼─────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-raspy-meadow-26349337 │ development │ false   │ 2023-11-28T19:19:11Z │ 2023-12-01T00:18:21Z │
+├──────────────────────────┼─────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-curly-bar-82389180    │ main        │ true    │ 2023-10-23T12:49:41Z │ 2023-12-01T00:18:21Z │
+└──────────────────────────┴─────────────┴─────────┴──────────────────────┴──────────────────────┘
+```
+
+### Using a named `context-file`
+
+Set the context to a different project in the same account:
+```bash
+neonctl set-context --project-id plain-waterfall-84865553 --context-file Documents/MyContext
+```
+List all branches using the `branches list` command. No need to specify the project since the context-file provides it.
+```bash
+neonctl branches list --context-file Documents/MyContext
+```
+The results show details for all branches in the `plain-waterfall-84865553` project:
+```bash
+┌─────────────────────────────┬─────────────┬─────────┬──────────────────────┬──────────────────────┐
+│ Id                          │ Name        │ Primary │ Created At           │ Updated At           │
+├─────────────────────────────┼─────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-soft-base-86343042       │ development │ false   │ 2023-11-21T18:41:47Z │ 2023-12-01T00:00:14Z │
+├─────────────────────────────┼─────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-young-bush-89857627      │ main        │ true    │ 2023-11-21T18:00:10Z │ 2023-12-01T03:33:53Z │
+├─────────────────────────────┼─────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-billowing-union-41102466 │ staging     │ false   │ 2023-11-21T18:44:22Z │ 2023-12-01T08:32:40Z │
+└─────────────────────────────┴─────────────┴─────────┴──────────────────────┴──────────────────────
+```
+
+<Admonition type="note">
+These two `branches list` commands demonstrate the use of different contexts in the same account. The hidden file context is set to `patient-frost-50125040` while the named `context-file` is set to `plain-waterfall-84865553`. These contexts operate independently. You can set as many `context-files` as you'd like, using unique names or in different directories, depending on your needs.
+</Admonition>
+
+### Setting context when creating a new project
+
+Let's say you want to create a new project called `MyLatest`. You can automatically set the project ID and primary branch ID  context at the same time as you create the project.
+
+```bash
+neonctl projects create --name MyLatest --set-context
+```
+
+This creates a hidden file with the following context details:
+
+```json
+{
+  "projectId": "quiet-water-76237589",
+  "branchId": "br-still-wind-46531853"
+}
+```
+You can now use any command that would normally require an additional `--project-id` or `branch` parameter and the command will default to this context.
```

Comment:
I would also add that: we don't save any confidential info, like auth tokens to the context file, so it's ok to commit it into git and share with your coworkers.

Reply:
Added admonition:
Neon does not save any confidential information to the context file (for example, auth tokens). You can safely commit this file to your repository or share with others.